src/framework/backend/udev.rs

```diff
@@ -267,9 +267,8 @@ struct OutputSurfaceData {
     surface: RenderSurface,
     _render_timer: RenderTimerHandle,
     fps: fps_ticker::Fps,
-    imgui: Option<imgui::SuspendedContext>,
-    imgui_pipeline: imgui_smithay_renderer::Renderer,
-
+    //imgui: Option<imgui::SuspendedContext>,
+    //imgui_pipeline: imgui_smithay_renderer::Renderer,
```

Comment:
Let's just remove this, if it is no longer used

Reply:
sure, forgot about this, fixed